### PR TITLE
Icon

### PR DESCRIPTION
**Pull request by tchiliali**
this is for icon

### CODE REVIEW

File: migcontrol/templates/base.html [hunk 1]
```diff
@@ -105,7 +105,10 @@
         </ul>
         <form class="search-form d-flex" action="{% url 'search' %}" method="GET">
           <input name="query" class="form-control me-2" type="search" placeholder="{% trans "Search" %}" aria-label="Search">
-          <button class="btn btn-success" type="submit">{% trans "Search" %}</button>
+
+
+          <button class="fa fa-search" style="color:red" type="submit" >{% trans "" %}</button>
```
Comment (benjaoming) [inline migcontrol/templates/base.html:110]:
```suggestion
          <button class="fa fa-search" style="color:red" type="submit" ></button>
```

**Review by benjaoming — CHANGES_REQUESTED** [review]
As mentioned in the former review, please remove this

File: migcontrol/templates/base.html (hunk 1 above)
Comment (tchiliali) [reply migcontrol/templates/base.html:110]: This will only affect line 110 right?

File: migcontrol/templates/base.html [hunk 2]
```diff
@@ -105,7 +105,10 @@
         </ul>
         <form class="search-form d-flex" action="{% url 'search' %}" method="GET">
           <input name="query" class="form-control me-2" type="search" placeholder="{% trans "Search" %}" aria-label="Search">
-          <button class="btn btn-success" type="submit">{% trans "Search" %}</button>
+
+
+          <button class="fa fa-search" style="color:red" type="submit" ></button>
```
Comment (benjaoming) [inline migcontrol/templates/base.html:110]: Hi Tchili, you have added `style="color:red"` which is an inline style. We cannot and should not use this. Can you remove it? Thanks.

**Review by benjaoming — CHANGES_REQUESTED** [review]

**Pull request merged**